app/script/view_model/panel/ConversationParticipantsViewModel.js

```diff
@@ -35,6 +35,7 @@ z.viewModel.panel.ConversationParticipantsViewModel = class ConversationParticip
       }
       return [];
     });
+    this.highlightedUsers = ko.observable([]);
```

Comment:
Do you really need the power of observables here? It seems to me that a native array would do the trick. Am I wrong?

Reply:
The renderer needs this observable.